dev-docs/Cloud-Env-Upgrade.md

```diff
@@ -0,0 +1,54 @@
+# Cloud Environment Upgrade Testing
+
+The [`Test Upgrade Environment`](https://github.com/elastic/cloudbeat/actions/workflows/upgrade-environment.yml) GitHub action automates the process of deploying a fully-featured cloud environment, pre-configured with all integrations (KSPM, CSPM, and D4C).
+It also facilitates the upgrade of the environment to a new version of the ELK stack and all installed agents, along with performing findings retrieval checks.
+
+
+## How to Run the Workflow
+
+Follow these steps to run the workflow:
+
+1. Go to [`Actions > Test Upgrade Environment`](https://github.com/elastic/cloudbeat/actions/workflows/upgrade-environment.yml).
+
+   ![Navigate to Actions](https://github.com/elastic/cloudbeat/assets/99176494/2686668f-7be6-4b55-a37b-e37426c1a0e1)
+
+2. Click the `Run workflow` button.
+
+   ![Run Workflow](https://github.com/elastic/cloudbeat/assets/99176494/902efe40-ed1b-4175-92a6-504439eb9e3d)
+
+3. Complete the required parameters:
+
+    - **`deployment_name`**: Name your environment (Allowed characters: a-z0-9 and `-`). For
+      instance: `john-8-11-0-nov1`.
+
+    - **`elk-stack-version`**: Specify the version of Elastic Cloud stack, either a SNAPSHOT or a build candidate (BC)
+      version. Check the available versions [here](https://artifacts-staging.elastic.co/dra-info/index.html).
+      For BC, enter only the version without additions/commit sha, e.g. `8.11.0`.
+      For SNAPSHOT, enter the full version, e.g. `8.12.0-SNAPSHOT`.
```

Comment:
It's not clear what this version is affecting in the context of the upgrade process. Is it the starting point from which the upgrade will take place, or is it the target stack version that the upgrade will aim to reach?

Reply:
Documentation is updated